src/AspNetCoreModuleV2/CommonLib/config_utility.h

```diff
@@ -0,0 +1,105 @@
+// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+#pragma once
+
+#include "stdafx.h"
+
+class ConfigUtility
```

Comment:
Not blocking this PR, but why isn't this part of shim config itself?

Reply:
Will be shared with request_handler_config eventually.

Reply:
Why?

Reply:
The handler settings section needs is meant to be read and used by the request handlers. It's only used by the shim now.